README.asciidoc

```diff
@@ -131,6 +134,13 @@ The plugin defines the following extension properties in the `docker` closure:
 |`apiVersion`    |String    |null                     |The remote API version. For most cases this can be left null.
 |=======
 
+[WARNING]
+====
+Default value of `docker.url` property is deprecated.
+It will be replaced to `unix:///var/run/docker.sock` in upcoming releases.
+Please adjust your build scripts accordingly.
```

Comment:
On Linux, one should enable tcp explicitly, since by default `Docker Engine` only listens to unix socket. And `docker-java` defaults to `unix://` as well.

Not sure, the plugin thusl provide own defaults in this case.